src/dune_rules/lib.mli

```diff
@@ -235,3 +235,5 @@ module Local : sig
 
   include Comparable_intf.S with type key := t
 end
+
+module Tbl : Hashtbl.S with type key := t
```

Comment:
I don't think you need this since our `Table` module is polymorphic. `let table = Table.create (module Lib)` should work